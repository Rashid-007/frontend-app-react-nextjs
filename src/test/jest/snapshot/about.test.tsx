import React from "react";
import renderer from "react-test-renderer";
import About from "../../../main/pages/about";

describe("About content test", () => {
    it("test link", () => {
        const about = renderer.create(<About/>).toJSON;
        expect(about).toMatchSnapshot();
    });
})