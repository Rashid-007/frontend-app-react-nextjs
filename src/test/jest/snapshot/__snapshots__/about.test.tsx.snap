// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About content test test link 1`] = `[Function]`;
